meadow-client/src/MainFrame.purs

```diff
@@ -78,7 +79,7 @@ emptyMarloweState :: MarloweState
 emptyMarloweState = { input: emptyInputData
                     , transaction: emptyTransactionData
                     , state: emptyState
-                    , blockNum: (fromInt 0)
+                    , blockNum: fromIntegral 0
```

Comment:
Top trivia: If there's a `Semiring` instance, you can just use `zero` here and you'll get the right type automatically. Useful when you know it should be conceptually zero, but you don't want to worry about marshalling types.

Reply:
nice